src/ui/components/CCDiceMenu.vue

```diff
@@ -298,10 +299,17 @@ export default Vue.extend({
   }),
   mounted() {
     this.reset()
+    if (this.autoroll) this.$nextTick(this.autoRoll)
   },
   watch: {
     menu() {
-      this.reset()
+      if (!this.menu || !this.autoroll) this.reset()
+    },
+    presetAccuracy() {
+      if (this.autoroll) {
+        this.accuracy=this.presetAccuracy
+        this.autoRoll()
```

Comment:
As you update accuracy and difficulty, this will automatically re-roll with the new accuracy and difficulty.

Reply:
https://user-images.githubusercontent.com/6234658/128186917-3fbf214d-125c-4c0f-93e6-c0d33fef5e90.mp4